utils.py

```diff
@@ -89,7 +95,11 @@ def get_vlan_tags_and_masks(tag_ranges: list[list[int]]) -> list[str]:
             divisor = max_power2_divisor(start)
             while divisor > limit - start:
                 divisor //= 2
-            masks_list.append(f"{start}/{4096-divisor}")
+            mask = 4096 - divisor
+            if mask == 4095:
+                masks_list.append(start)
```

Comment:
I cannot figure this case out, although I have not checked wireshark. For any flow with `vlan/mask` that has a 4095 mask, the string is replaced by an interger `vlan` when the flow is in the `switch`. For example `"101/4095"` is replaced by `101`.
The flow is sent correctly from `flow_manager`, [code](https://github.com/kytos-ng/flow_manager/blob/58f11e8680132dffc991e164de9894d663361423/main.py#L782), but received as string in `of_core`, [code](https://github.com/kytos-ng/of_core/blob/4503554180d51584e7adf5b0136e47a2fd8dddf1/main.py#L195) (check on reply.body).
For now, I am treating this case as a feature and not a bug since it does not interrupt any of `vlan_range` development. But probably not final.